src/js/fluxcontroller.js

```diff
@@ -561,9 +577,10 @@ define(function (require, exports, module) {
      */
     FluxController.prototype._applyAction = function (action, actionReceiver, params, parentActionName) {
         var flux = this.flux,
-            lockUI = action.lockUI,
-            post = action.post,
-            modal = action.modal || false,
+            actionObject = action.action,
```

Comment:
Also, if (as discussed above) it _is_ possible for `action.action` to be falsey, then this could cause an error below.